### PR TITLE
Rewrite lxd_exporter

### PR DESCRIPTION
**Pull request by nieltg**
Rewritten `lxd_exporter` is equipped with tests also, and better error messages.

Corresponding issue: #7.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 viveksing
+Copyright (c) 2019 Daniel Pintara
```
Comment (ekarisky) [inline LICENSE]: Please change this @nieltg

File: README.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
-# lxdexporter_golang
+# lxd_exporter
 
-Set Environment variable LXD_SOCKET to use different path ( default is /var/lib/lxd/unix.socket )
+[![Build Status](https://travis-ci.org/nieltg/lxd_exporter.svg?branch=master)](https://travis-ci.org/nieltg/lxd_exporter)
```
Comment (ekarisky) [inline README.md]: Please change this url

File: pkg/metrics/collector_test.go [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+	go func() {
+		for range ch {
+		}
+	}()
+
+	NewCollector(logger, server).Collect(ch)
+	close(ch)
+}
+
+func Example_collector_Collect_containerNamesError() {
```
Comment (ekarisky) [inline pkg/metrics/collector_test.go:40]: Can we change func name using standard go convention?

**Review by ekarisky — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (nieltg) [reply LICENSE]: I had changed this. Please take a look.

File: README.md (hunk 2 above)
Comment (nieltg) [reply README.md]: I had changed these URLs. Please take a look.

File: pkg/metrics/collector_test.go (hunk 3 above)
Comment (nieltg) [reply pkg/metrics/collector_test.go:40]: This link refers to the standard Go convention: https://golang.org/pkg/testing/#hdr-Examples. The documentation doesn't state about the naming convention of private members, so I add underscore.

Adding underscore is needed for Go to recognize test functions. This issue has been acknowledged by @ekarisky from personal chat.

**Review by ekarisky — APPROVED** [review]

**Pull request merged**